Added support for Affinity

dylanmartinezs opened the pull request:
Added tested support for:
- Affinity Designer
- Affinity Photo
- ~~Adobe Acrobat CC~~
- Affinity Publisher (not tested, but it should work as the others)

dmartinez301299 committed:
Added support for:
- Affinity Designer
- Affinity Photo
- Adobe Acrobat CC

dmartinez301299 committed:
Merge branch 'main' of https://github.com/rataman098/winapps into main

dmartinez301299 committed:
Removed Acrobat Reader CC dupe

mwanago commented:
Even though Affinity Photo starts fine for me, it always crashes when trying to create or open files

Phosphorus-M commented:
I suggest change the files icons, because the real icon have copyright.

rataman098 commented:
> Even though Affinity Photo starts fine for me, it always crashes when trying to create or open files

I don't know what may be the problem, I've been able to run it without any problems so far 🤔

[Affinity Photo running in UbuntuDDE](https://forum.affinity.serif.com/uploads/monthly_2020_11/1266605349_ScreenCapture_20201119174919.png.c3e59d9e3792f894dbd8549a2dab4ce5.png)

rataman098 commented:
> I suggest change the files icons, because the real icon have copyright.

Hmmmm, I'll see what I can do about that

Commit:
Added support for Affinity Publisher and:
- Added custom icon for Designer
- Added custom icon for Photo
- Added custom icon for Publisher

dylanmartinezs reviewed (commented):
Good me!

rataman098 commented:
> I suggest change the files icons, because the real icon have copyright.

As I didn't find any free icon, I've decided to design my own :P
You're free to use these icons however you want, credit is not necessary 😄

KernelGhost commented:
@dylanmartinezs Thank you for your contribution. Unfortunately, this repository has seen little activity and is now quite outdated. Development for WinApps has moved to a new location: https://github.com/winapps-org/winapps. We would be more than happy to review and merge your changes if you would like to submit a new pull request to the active repository. Thank you!
